samples/python/winml_tutorial/pywinrt_output/pyrt/src/py.Windows.Foundation.Collections.h

```diff
@@ -0,0 +1,2760 @@
+// WARNING: Please don't edit this file. It was generated by Python/WinRT
+
+#pragma once
+
+#include "pybase.h"
+
+#if __has_include("py.Windows.Foundation.h")
+#include "py.Windows.Foundation.h"
+#endif
+
+#include <winrt/Windows.Foundation.Collections.h>
+
+template <typename K, typename V>
+struct pyMapChangedEventHandler
+{
+    static winrt::Windows::Foundation::Collections::MapChangedEventHandler<K, V> get(PyObject* callable)
+    {
+        py::throw_if_pyobj_null(callable);
+        
+        py::delegate_callable cb{ callable };
```

Comment:
nit: short names are ambiguous (cb: count of bytes?)

Reply:
Please comment on #217